### DISCUSSION TITLE
Question about LoTW upload.

### DISCUSSION THREAD

**Discussion by R1BLH** [wavelog/wavelog#919] [Q&A] [answered]
I have found that station locations with "/P" are excluded from LoTW upload.

I have 6 stations in my log, one is "/0" -- from another DXCC (Asian Russia) with separate LoTW certificate, the rest are from the one DXCC (European Russia) covered by my main DXCC certificate. Two of them are from different portable locations (with /P in the callsign).
![изображение](https://github.com/user-attachments/assets/ae3a9be3-b2f5-4ccf-b5ea-8bba63f236ce)

But I have found that stations with /P callsign do not synchronise with LoTW.
![изображение](https://github.com/user-attachments/assets/d3cd588b-a0d7-4fdf-a9e6-43da24deb480)

What could be the reason for this situation? Should I request a separate LoTW certificate for /P (/M) callsign?

**Comment by HB9HIL** [comment, marked as answer]
You need a LoTW Cert for every call
R1BLH is a different call then R1BLH/P

<img width="419" alt="image" src="https://github.com/user-attachments/assets/4cb1702f-2170-4802-93ff-ff2bd36a5755">
